lapce-app/src/window_tab.rs

```diff
@@ -437,9 +439,18 @@ impl WindowTabData {
         let panel = workspace_info
             .as_ref()
             .map(|i| {
-                let panel_order = db
+                let mut panel_order = db
                     .get_panel_orders()
                     .unwrap_or_else(|_| i.panel.panels.clone());
+                if !panel_order
+                    .iter()
+                    .any(|x| x.1.iter().any(|x| *x == PanelKind::CallHierarchy))
+                {
+                    panel_order
+                        .entry(PanelPosition::BottomLeft)
+                        .or_default()
+                        .push_back(PanelKind::CallHierarchy);
+                }
```

Comment:
`db.get_panel_orders()` should have taken care of this.

Reply:
For a newly upgraded lapce, there will be problems. At this time, the database does not have this type, which will cause the ui to fail to display.I had this problem when I was developing it.

Reply:
Or don't even think about this scenario, you decide?